core/iwasm/aot/aot_runtime.c

```diff
@@ -2264,7 +2264,7 @@ aot_get_module_inst_mem_consumption(const AOTModuleInstance *module_inst,
         mem_conspn->memories_size +=
             mem_inst->num_bytes_per_page * mem_inst->cur_page_count;
         mem_conspn->app_heap_size =
-            mem_inst->heap_data_end.ptr - mem_inst->heap_data.ptr;
+            (uint8_t *) mem_inst->heap_data_end.ptr - (uint8_t *) mem_inst->heap_data.ptr;
```

Comment:
Hi, @Paril, we defined uint8, int64, uint64 types in the platform header file, and usually use them inside runtime instead of uint8_t, int64_t, uint64_t and so on. Could you please modify the related changes so that the coding style is unified?

Reply:
Oops, will do.